igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/model/GenericBuild.groovy

```diff
@@ -16,23 +16,37 @@
 
 package com.netflix.spinnaker.igor.build.model
 
+import com.fasterxml.jackson.annotation.JsonInclude
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
 class GenericBuild {
     boolean building
     String fullDisplayName
     int number
-    int duration
+    Integer duration
```

Comment:
The reason I changed duration from int to Integer is that an integer does not report as 0 if it is unset. The change back to int broke the build because a test expect only building and number to be set
```
    |        |               {"building":false,"number":123(,"duration":0)}
    |        |               {"building":false,"number":123(-------------)}
    |        {"building":false,"number":123,"duration":0}
```